.travis.yml

```diff
@@ -1,14 +1,20 @@
 language: node_js
 node_js:
-  - "8"
-  - "9"
-  - "10"
+- '8'
+- '9'
+- '10'
 script:
-  - yarn test
-
+- yarn test
 after_success: yarn coverage
-
 notifications:
   email:
     on_success: never
     on_failure: always
+deploy:
+  provider: npm
+  email: $NPM_EMAIL
+  api_key: $NPM_KEY
```

Comment:
These are defined in the Travis environment. Currently using my credentials. Feel free to update if you like.

Reply:
Seems fine. Do you mind if I switch it to publishing as lifeomicdev?  I should have the credentials needed to do that.

Reply:
I modified the project configuration and I'll publish a new version as a test after merging this